client/secureclient/src/main/java/com/linkedin/openhouse/client/ssl/WebClientFactory.java

```diff
@@ -24,6 +24,8 @@ public abstract class WebClientFactory {
   private static final String HTTP = "http";
   private static final String HTTPS = "https";
   private static final String SESSION_ID = "session-id";
+
+  private static final String HTTPHEADER_CLIENT_NAME = "X-Client-Name";
```

Comment:
unfortunately can't use constant from services.common.Constants, so defining this string key twice in this PR.